core/scripts/functions/templates/oracle.toml

```diff
@@ -15,10 +15,11 @@ observationSource = """
     decode_log   [type="ethabidecodelog" abi="OracleRequest(bytes32 indexed requestId, address requestingContract, address requestInitiator, uint64 subscriptionId, address subscriptionOwner, bytes data)" data="$(jobRun.logData)" topics="$(jobRun.logTopics)"]
     decode_cbor  [type="cborparse" data="$(decode_log.data)"]
     run_computation    [type="bridge" name="ea_bridge" requestData="{\\"requestId\\": $(jobRun.meta.oracleRequest.requestId), \\"jobName\\": $(jobSpec.name), \\"subscriptionOwner\\": $(decode_log.subscriptionOwner), \\"subscriptionId\\": $(decode_log.subscriptionId), \\"data\\": $(decode_cbor)}"]
-    parse_result [type=jsonparse data="$(run_computation)" path="data,result"]
-    parse_error  [type=jsonparse data="$(run_computation)" path="data,error"]
+    parse_result  [type=jsonparse data="$(run_computation)" path="data,result"]
+    parse_error   [type=jsonparse data="$(run_computation)" path="data,error"]
+    parse_domains [type=jsonparse data="$(run_computation)" path="data,domains" lax=true]
 
-decode_log -> decode_cbor -> run_computation -> parse_result -> parse_error
+decode_log -> decode_cbor -> run_computation -> parse_result -> parse_error -> parse_domains
```

Comment:
We now hardcode the whole job observationSource so a change here is not necessary.

Reply:
I wanted to keep our scripts matching the changes.